### DISCUSSION TITLE
Suply is too slow to upload/prepare aab

### DISCUSSION THREAD

**Discussion by ospfranco** [fastlane/fastlane#21182] [Q&A]
I'm running Fastlane on a m2 max machine, the compilation is very fast, it takes at most a minute to build my app. However, when uploading to the play store the supply step `preparing aab at path` is very slow... sometimes 3 minutes of more.

Is anyone experiencing this as well? Is there any reason for it?

**Comment by axi92** [comment]
I guess like apple, google is doing some internal verifications before fastlane returns and that takes some time.
I guess fastlane is just waiting for the response from the google api.
